Tasks/XcodeV5/xcodeutils.ts

```diff
@@ -205,6 +205,19 @@ export function uploadLogFile(logFile: string) {
     }
 }
 
+export function publishXcodeTelemetry(telemetryData: { [key: string]: any; }) {
+    // This check can be removed when we compile with --strict-null-check.
+    if (!telemetryData) {
+        return;
+    }
+    try {
+        tl.assertAgent('2.120.0');
+        console.log('##vso[telemetry.publish area=Tasks.CrossPlatform;feature=Xcode]' + JSON.stringify(telemetryData));
```

Comment:
Can we change the `area` from `Tasks.CrossPlatform` to `TaskHub`?  That's the area to which other tasks log telemetry. 
 It would allow querying all categories of tasks at once.  We would likewise change `Tasks.CrossPlatform` to `TaskHub` in the only other place it's used: `Tasks/Common/java-common/java-common.ts` line [65](https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/Common/java-common/java-common.ts#L65).

Reply:
Done. I created a separate PR for java-common.ts: #8023